src/components/api-tokens/views.tsx

```diff
@@ -0,0 +1,309 @@
+import moment from 'moment';
+import React, { ReactElement } from 'react';
+
+import { DATE_TIME, SLUG_REGEX } from '../../layouts';
+import { IOrganization, IOrganizationUserRoles } from '../../lib/cf/types';
+import { IUaaUser } from '../../lib/uaa';
+import { RouteLinker } from '../app';
+
+interface IListTokensProperties {
+  readonly linkTo: RouteLinker;
+  readonly organization: IOrganization;
+  readonly tokens: ReadonlyArray<IOrganizationUserRoles>;
+}
+
+interface IExposeSecretProperties {
+  readonly linkTo: RouteLinker;
+  readonly organization: IOrganization;
+  readonly tokenKey: string;
+  readonly tokenSecret: string;
+  readonly userGUID: string;
+}
+
+interface IConfirmActionProperties {
+  readonly action: 'rotate' | 'revoke';
+  readonly csrf: string;
+  readonly linkTo: RouteLinker;
+  readonly organization: IOrganization;
+  readonly token: IUaaUser;
+}
+
+interface ICreateAPITokenProperties {
+  readonly csrf: string;
+  readonly error?: boolean;
+  readonly organization: IOrganization;
+  readonly values?: {
+    readonly name?: string;
+  };
+}
+
+export function ListTokens(props: IListTokensProperties): ReactElement {
+  return <div className="govuk-grid-row">
+    <div className="govuk-grid-column-full">
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <h1 className="govuk-heading-l">API Tokens</h1>
+
+          <p className="govuk-body">
+            API Tokens can be used as robot users for tooling such as Continuous Delivery. For instance, a GitHub
+            Actions pipeline capable of deploying an applciation after passing defined set of tests.
+          </p>
+          <p className="govuk-body">
+            Each API Token has it&apos;s own set of permissions much like the individual users.
+          </p>
+        </div>
+
+        <div className="govuk-grid-column-one-third text-right">
+          <a
+            href={props.linkTo('admin.organizations.tokens.create', {
+              organizationGUID: props.organization.metadata.guid,
+            })}
+            role="button"
+            draggable="false"
+            className="govuk-button"
+            data-module="govuk-button"
+          >
+            Create a new API Token
+          </a>
+        </div>
+      </div>
+
+      <h2 className="govuk-heading-m">Current API Tokens</h2>
```

Comment:
can we only have the table if there are any tokens?
If there are no token either have text 'There are no tokens' or better word, or not render the heading either, whichever one is easietr